code/game/objects/machinery/mortar.dm

```diff
@@ -39,6 +39,8 @@
 		/obj/item/mortal_shell/plasmaloss,
 	)
 
+	var/tally_type = 1	//Used for round stats. 1 is mortar, 2 is howitzer
```

Comment:
you need autodoc, and you need to use define
///Used for round stats
var/tally_type = MORTAR_ROUND_TALLY